Firmware/ultralcd.cpp

```diff
@@ -857,16 +857,16 @@ void lcd_commands()
                 lay1cal_meander(layer_height, extrusion_width);
                 break;
             case 6:
-                lay1cal_square(0, layer_height, extrusion_width);
+                lay1cal_square(layer_height, extrusion_width);
                 break;
             case 5:
-                lay1cal_square(1, layer_height, extrusion_width);
+                lay1cal_square(layer_height, extrusion_width);
                 break;
             case 4:
-                lay1cal_square(2, layer_height, extrusion_width);
+                lay1cal_square(layer_height, extrusion_width);
                 break;
             case 3:
-                lay1cal_square(3, layer_height, extrusion_width);
+                lay1cal_square(layer_height, extrusion_width);
```

Comment:
Surely this can be optimised now :)? `layer_height` and `extrusion_width` are defined as constants.

Prehaps:


```c
case 6:
case 5:
case 4:
case 3:
    lay1cal_square(layer_height, extrusion_width);
    break;
```

Reply:
Indeed. But we can assume the nozzle diameter wont change during first layer calibration.

Reply:
No it doesn't change.

Reply:
Made a PR: https://github.com/3d-gussner/Prusa-Firmware/pull/97

Reply:
merged